floors/fetcher_test.go

```diff
@@ -1219,7 +1219,6 @@ func TestPriceFloorFetcherSubmitFailed(t *testing.T) {
 }
 
 func getRandomNumber() int {
-	rand.Seed(time.Now().UnixNano())
```

Comment:
What version of Go makes this unnecessary? We still support 1.19.

Reply:
go.mod has  `go 1.20` so I thought it is the minimal supported version. I will revert and add a `//nolint` comment.

Reply:
done